python/kfserving/kfserving/api/kf_serving_client.py

```diff
@@ -292,34 +298,33 @@ def delete(self, name, namespace=None, version=constants.KFSERVING_VERSION):
                 version,
                 namespace,
                 constants.KFSERVING_PLURAL,
-                name,
-                client.V1DeleteOptions())
+                name)
         except client.rest.ApiException as e:
             raise RuntimeError(
                 "Exception when calling CustomObjectsApi->delete_namespaced_custom_object:\
                  %s\n" % e)
 
-
-    def is_isvc_ready(self, name, namespace=None): #pylint:disable=inconsistent-return-statements
+    def is_isvc_ready(self, name, namespace=None, ):  # pylint:disable=inconsistent-return-statements
         """
         Check if the inference service is ready.
         :param name: inference service name
         :param namespace: defaults to current or default namespace
         :return:
         """
-        kfsvc_status = self.get(name, namespace=namespace, version=constants.KFSERVING_V1BETA1_VERSION)
+        kfsvc_status = self.get(name, namespace=namespace,
+                                version=constants.KFSERVING_V1BETA1_VERSION)
         status = 'Unknown'
         for condition in kfsvc_status['status'].get('conditions', {}):
             if condition.get('type', '') == 'Ready':
                 status = condition.get('status', 'Unknown')
                 return status.lower() == "true"
         return False
 
-
-    def wait_isvc_ready(self, name, namespace=None, #pylint:disable=too-many-arguments
+    def wait_isvc_ready(self, name, namespace=None,  # pylint:disable=too-many-arguments
                         watch=False,
                         timeout_seconds=600,
-                        polling_interval=10):
+                        polling_interval=10,
+                        version=constants.KFSERVING_VERSION):
```

Comment:
I think get operations should always use v1beta1 version, so we probably do not need this argument. Only when you create or update user can still do v1alpha2.

Reply:
I updated the default version for get operation to v1beta1, and keep that as option since I guess user may get the version in old KFS env. we can remove that next release. Thanks.

Reply:
Also updated here to remove version arg.